www/cordova/apphost.js

```diff
@@ -234,24 +234,18 @@ define(['appStorage', 'browser'], function (appStorage, browser) {
             return new Promise(function (resolve, reject) {
 
                 document.addEventListener("deviceready", function () {
+                    // remove special characters
+                    var deviceName = MainActivity.getDeviceModel().replace(/[^\w\s]/gi, '');
+                    var deviceId = MainActivity.getDeviceId();
 
-                    var name = "Emby for Android Mobile";
-
-                    // Remove special characters
-                    var cleanDeviceName = MainActivity.getDeviceModel().replace(/[^\w\s]/gi, '');
```

Comment:
Just a little concerned on this part - if I read this correctly, they’re sanitizing the device name before it gets sent to the server, right? I haven’t looked at this part of the server code (yet), but I wonder if there’s an issue with special characters.

Reply:
Yeah it is still included, just renamed some variables here.